src/main/kotlin/org/jetbrains/bio/util/StringUtil.kt

```diff
@@ -1,8 +1,18 @@
 package org.jetbrains.bio.util
 
 import org.apache.commons.math3.util.Precision
+import java.io.File
 import java.util.*
 
+fun String.separatorsToSystem(): String {
+    return if (File.separatorChar == '\\') {
+        // From Windows to Linux/Mac
+        this.replace('/', File.separatorChar)
```

Comment:
Is it an error in a comment?  In line 10 file separator is windows like, e.g. `\`, so seems it replaces `from Linux/Max to Windows`

Reply:
Tests are missing in `StringUtilTest`. As usual, unit-tests are obligatory for public API even if the method looks simple. To make tests platfrorm-independent, I suggest to extract `File.separatorChar` into a method argument with default value `File.separatorChar`. So you will be able in test emulate both cases, when separatorChar is `/` and when it is '\'